test/indexer/test_projects.py

```diff
@@ -48,6 +47,7 @@ def setUpClass(cls):
     def tearDownClass(cls):
         super().tearDownClass()
 
+    @unittest.skip
```

Comment:
Needs ticket.

Reply:
This line shouldn't have been committed. My mistake.